docker/ravendb-ubuntu1604/Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN apt-get update \
     && apt-get install -y \
     && apt-get install --no-install-recommends curl bzip2 libunwind8 libicu55 libcurl3 ca-certificates jq -y
 
-ENV DATA_DIR=/databases CUSTOM_CONFIG_FILE='' PUBLIC_SERVER_URL='' PUBLIC_TCP_SERVER_URL='' CERTIFICATE_PATH='' CERTIFICATE_PASSWORD='' CERTIFICATE_PASSWORD_FILE='' UNSECURED_ACCESS_ALLOWED='' LOGS_MODE='' RAVEN_AUTO_INSTALL_CA='true' RAVEN_IN_DOCKER='true'
+ENV RAVEN_DataDir=/databases RAVEN_AUTO_INSTALL_CA='true' RAVEN_IN_DOCKER='true'
```

Comment:
`RAVEN_AUTO_INSTALL_CA` / `RAVEN_IN_DOCKER` are required?

Reply:
First one was used by @iftahbe to auto install certs, not sure if still relevant. Not sure about RAVEN_IN_DOCKER right now - it was used to update cli args after setup. Let's leave it here for now.

Reply:
Both are not relevant anymore.